### ISSUE TITLE
Intellisense missing for __typename in Playground

### ISSUE THREAD

**Issue by kbrandwijk** [graphcool/console#1194]
**Do you want to request a *feature* or report a *bug*?**  
Bug/feature

**What is the current behavior?**
In the Playground, there's no intellisense for the query result field `__typename`. Only after typing it manually, and seeing no 'red line' underneath, I know that I have apparently typed it correctly (was it a small n, or a capital N?). 

**What is the expected behavior?**
It would be helpful if the field showed up in the intellisense popup.

**Comment by kbrandwijk** [comment]
Also reported with graphiql, I guess this will end up on hold until they fix it... https://github.com/graphql/graphiql/issues/535

**Comment by timsuchanek** [comment]
Thanks for reporting. Yes, we have to wait for GraphiQL to fix that
